test/reindexer/test_qubit_reindexer.py

```diff
@@ -0,0 +1,55 @@
+from typing import List
```

Comment:
For completeness, it may be wise to add a measurement test after reindexing.

Reply:
Thanks! I've changed the `replacement_gates_4` input so that it uses all the visited methods in `_QubitReindexer`.